boot/boot.cmd

```diff
@@ -133,16 +133,15 @@ elif test "${pinebook_lcd_mode}" = "batch2"; then
 	fdt set /soc@01c00000/lcd0@01c0c000 lcd_vspw "4"
 elif test "${pinebook_lcd_mode}" = "1080p"; then
 	echo "Fixing LCD parameters to use Pinebook 1080p"
-	#fdt set /soc@01c00000/lcd0@01c0c000 lcd_if "0"
-	fdt set /soc@01c00000/lcd0@01c0c000 lcd_x "1920"
-	fdt set /soc@01c00000/lcd0@01c0c000 lcd_y "1080"
-	fdt set /soc@01c00000/lcd0@01c0c000 lcd_dclk_freq "139"
-	fdt set /soc@01c00000/lcd0@01c0c000 lcd_hbp "112"
-	fdt set /soc@01c00000/lcd0@01c0c000 lcd_hspw "32"
-	fdt set /soc@01c00000/lcd0@01c0c000 lcd_ht "2080"
-	fdt set /soc@01c00000/lcd0@01c0c000 lcd_vbp "28"
-	fdt set /soc@01c00000/lcd0@01c0c000 lcd_vspw "5"
-	fdt set /soc@01c00000/lcd0@01c0c000 lcd_vt "1111"
+	fdt set /soc@01c00000/lcd0@01c0c000 lcd_x "<1920>"
```

Comment:
Why we need '<'?

Reply:
Btw. you should modify `blobs/boot.cmd` which is copied to `boot/boot.cmd`.

Reply:
We need <> because otherwise it sets it as text value, not as integer. Display driver expects integers.

Reply:
Yes. I noticed that.